### DISCUSSION TITLE
Zeka kurulu sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#150] [Zeka Sorusu]
Sonsuzus'da 10 fizikçi, 8 matematikçi ve de 4 bilişimci üye arasında 5 kişilik bir yönetim kurulu oluşturulmak isteniyor.
a- En az bir fizikçinin olduğu kaç farklı kurul oluşturulur?
b- Matematikçi sayısının fizikçi sayısından fazla, fizikçi sayısının bilişimci sayısından fazla olduğu kaç farklı kurul oluşturulur?

**Comment by Sadik1Dev** [comment]
a) 1 fizikçi olursa kalanların 4 üyeliğe paylaşım olasılığı= 21x20x19x18= 143640
b)Eğer kurul içinde bütün branşların olması zorunlu ise olasılık sıfırdır, bilişimcinin olamadığı durumda ise oluşabilecek farklı olasılık sayısı:
47.040
